### ISSUE TITLE
Problem with reading ssh key?

### ISSUE THREAD

**Issue by akrabat** [rlerdorf/php7dev#9]
When I try to get this running on my Mac, I get this error:

/www/php7dev/scripts/php7dev.rb:41:in `read': No such file or directory - /Users/rob/.ssh/id_rsa.pub (Errno::ENOENT)

The file exists and my user has read access to it. 

Any ideas on how to get around this problem?

Full output:

```
rob@swiftsure /www $ git clone https://github.com/rlerdorf/php7dev.git
Cloning into 'php7dev'...
remote: Counting objects: 120, done.
remote: Total 120 (delta 0), reused 0 (delta 0), pack-reused 120
Receiving objects: 100% (120/120), 21.89 KiB | 0 bytes/s, done.
Resolving deltas: 100% (63/63), done.
Checking connectivity... done.
rob@swiftsure /www $ cd php7dev/
rob@swiftsure /www/php7dev  (master)$ vagrant up
/www/php7dev/scripts/php7dev.rb:41:in `read': No such file or directory - /Users/rob/.ssh/id_rsa.pub (Errno::ENOENT)
    from /www/php7dev/scripts/php7dev.rb:41:in `block in configure'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/plugins/kernel_v2/config/vm_provisioner.rb:72:in `call'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/plugins/kernel_v2/config/vm_provisioner.rb:72:in `add_config'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/plugins/kernel_v2/config/vm.rb:311:in `provision'
    from /www/php7dev/scripts/php7dev.rb:39:in `configure'
    from /www/php7dev/Vagrantfile:18:in `block in <top (required)>'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/config/v2/loader.rb:37:in `call'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/config/v2/loader.rb:37:in `load'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/config/loader.rb:103:in `block (2 levels) in load'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/config/loader.rb:97:in `each'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/config/loader.rb:97:in `block in load'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/config/loader.rb:94:in `each'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/config/loader.rb:94:in `load'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/vagrantfile.rb:28:in `initialize'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/environment.rb:691:in `new'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/environment.rb:691:in `vagrantfile'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/environment.rb:441:in `host'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/environment.rb:207:in `block in action_runner'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/action/runner.rb:33:in `call'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/action/runner.rb:33:in `run'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/environment.rb:428:in `hook'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/environment.rb:673:in `unload'
    from /opt/vagrant/bin/../embedded/gems/gems/vagrant-1.7.2/bin/vagrant:177:in `ensure in <main>'
    from /opt/vagrant/bin/../embedded/gems/gems/vagrant-1.7.2/bin/vagrant:177:in `<main>'
rob@swiftsure /www/php7dev  (master)$ 
```

**Comment by rlerdorf** [comment]
Is this still happening with version 0.0.8?

**Comment by akrabat** [comment]
Nope. It's now fixed. 

Thanks.

**Comment by akrabat** [comment]
_sigh_

This is what happens when I don't pay attention. Sorry.

yes - it's still happening.

**Comment by rlerdorf** [comment]
Should be fixed by 29fbd4e20e847b2b007843731a11c1f62681ba10
